views/js/qtiRunner/core/Renderer.js

```diff
@@ -70,6 +71,10 @@ define([
         return valid;
     };
 
+    var _getThemeData = function(){
```

Comment:
this wrapper is unnecessary, why don't you call just  `var themeData = themesHelper.getCurrentThemeData();`  ? (and by convention we don't prefix with underscore)

Reply:
Removed